### PR TITLE
Added information

### PR DESCRIPTION
**Pull request by gavins13**
Added information about initial_state for Bidirectional RNNs

### CODE REVIEW

**Comment by gavinlive** [comment]
Note: just added comments. No code changes.

File: tensorflow/python/keras/layers/wrappers.py [hunk 1]
```diff
@@ -361,6 +361,10 @@ class Bidirectional(Wrapper):
   Call arguments:
     The call arguments for this layer are the same as those of the wrapped RNN
       layer.
+    Beware that when passing the `initial_state` argument during the call of this
+    layer, the first N elements in the `initial_state` list will be passed to the
```
Comment (qlzh727) [inline tensorflow/python/keras/layers/wrappers.py]: I think the N is the major word here and doesn't carry much meaning. How about first half and second half?

**Comment by gbaned** [comment]
@gavinlive Can you please check reviewer comments and keep us posted. Thanks!

**Review by gavins13 — COMMENTED** [review]
Changes made according to reviewer comments.

**Review by qlzh727 — APPROVED** [review]

**Comment by gbaned** [comment]
@gavinlive Can you please address Ubuntu Sanity errors? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
